src/azure-cli/azure/cli/command_modules/servicefabric/operations/managed_node_types.py

```diff
@@ -226,6 +232,47 @@ def add_vm_extension(cmd,
         raise
 
 
+def update_vm_extension(cmd,
+                        client,
+                        resource_group_name,
+                        cluster_name,
+                        node_type_name,
+                        extension_name,
+                        type_handler_version=None,
+                        force_update_tag=None,
+                        auto_upgrade_minor_version=None,
+                        setting=None,
+                        protected_setting=None,
+                        provision_after_extension=None,
+                        setup_order=None):
+    try:
+        node_type: NodeType = client.node_types.get(resource_group_name, cluster_name, node_type_name)
+        existing_extension = find_in_collection(node_type, 'vm_extensions', 'name', extension_name)
+
+        if existing_extension is None:
+            logger.error('Extension %s does not exist.', extension_name)
+            return None
+
+        newExtension = VMSSExtension(name=extension_name,
```

Comment:
suggest to change variable name to updatedExtension

Reply:
updated

Reply:
still see newExtension instead of updatedExtension